lib/calendarium-romanum/day.rb

```diff
@@ -12,6 +12,9 @@ def initialize(date: nil, season: nil, season_week: nil, celebrations: nil, vesp
       @vespers = vespers
     end
 
+    def succ
```

Comment:
This method isn't used anywhere, isn't covered by specs and is completely wrong. A `Day` has no clue about it's neighbours - only `Calendar` can say what a `Day` will be like - and two different `Day`s never share the same celebrations.